bb/validations.go

```diff
@@ -81,7 +83,12 @@ func bbValidateTitleInstructions(b interface{}) error {
 func bbValidateTitleDocumentNumber(b interface{}) error {
 	switch t := b.(type) {
 	case *models.BoletoRequest:
-		return t.Title.ValidateDocumentNumber()
+		maxAllowedLength := 15
+		if len(t.Title.DocumentNumber) > maxAllowedLength {
+			message := fmt.Sprintf("O campo documentNumber ultrapassou o limite permitido de %d caracteres", maxAllowedLength)
```

Comment:
@brunoglm Adicionaria a palavra `título` porque tem vários DocumentNumber no contrato (Buye, Recipient) ... Só pra ficar mais claro
```suggestion
			message := fmt.Sprintf("O campo documentNumber do título ultrapassou o limite permitido de %d caracteres", maxAllowedLength)
```

Reply:
Done 😀